test/distributions/test_distributions.py

```diff
@@ -4048,6 +4048,11 @@ def test_continuous_bernoulli_shape_tensor_params(self):
         self.assertRaises(ValueError, continuous_bernoulli.log_prob, self.tensor_sample_2)
         self.assertEqual(continuous_bernoulli.log_prob(torch.ones(3, 1, 1)).size(), torch.Size((3, 3, 2)))
 
+    def test_mixture_same_family_shape(self):
+        mix_distribution = Categorical(torch.ones([3,1,3]))
+        component_distribution = torch.distributions.Normal(torch.zeros([3,3,3]), torch.ones([3,3,3]))
+        gmm = MixtureSameFamilyFixed(mix, comp)
```

Comment:
whoops looks like you forgot to rename these variables

Reply:
...My God. This is what I get for not testing the exact same text I commit...chuckleheads like me are why we _need_ devops.